databook/physiology/ephys/np-ultra/npultra-psychedelics-stimuli.md

```diff
@@ -0,0 +1,14 @@
+# NP Ultra & Psychedelics Stimuli and Optotagging protocol
+
+## Spontaneous Activity
```

Comment:
add a link to the entries on the passive visual stimuli sections. Even if they're identical content.